everware/spawner.py

```diff
@@ -412,6 +412,7 @@ def get_env(self):
             env.update({
                 'JPY_GITHUBURL': self.repo_url_with_token,
                 'JPY_REPOPOINTER': self.commit_sha,
+                'API': 'api_from_openml'
```

Comment:
hmm, what `api_from_openml` should mean? the idea was that user (or even redirect from openml website) can supply his own unique openml key. here you just add a contant. how can it help?